framework/Models/Model.py

```diff
@@ -133,7 +133,11 @@ class cls.
   validateDict['Optimizer'][0]['class'       ] ='Optimizers'
   validateDict['Optimizer'][0]['required'    ] = False
   validateDict['Optimizer'][0]['multiplicity'] = 1
-  validateDict['Optimizer'][0]['type']         = ['SPSA','FiniteDifference','ConjugateGradient','SimulatedAnnealing']
+  validateDict['Optimizer'][0]['type']         = ['SPSA',
+                                                  'FiniteDifference',
+                                                  'ConjugateGradient',
+                                                  'SimulatedAnnealing',
+                                                  'GeneticAlgorithm']
```

Comment:
Adding GeneticAlgorithm probably should wait until GA is merged.

Reply:
addressed